src/client/tweetAnalysis.js

```diff
@@ -154,9 +154,32 @@ const wordCountToSortedList = (wordCounts) => {
   return result.sort((wordCount1, wordCount2) => (wordCount2.count - wordCount1.count));
 };
 
+/**
+ * Give a feed of Tweets, decide if the feed is still relevant.
+ * If a feed had less than 100 results, or more than 5 results are
+ * over 10 minutes old, it is considered not relevant.
+ * @param feedResults
+ * @returns {boolean}
+ */
 export const doesFeedHaveUsefulResults = (feedResults) => {
-  // TODO: Make this more useful
-  return feedResults.data.count > 10;
+  const now = new Date().getTime();
+  const tenMinutes = 1000 * 60 * 10;
+  const xMinutesAgo = now - tenMinutes;
+
+  const maxOldPosts = 5;
+
+  let outOfDateCount = 0;
+  for (const result in feedResults.data) {
+    const test = new Date(data.data.date).getTime();
+    if (test < xMinutesAgo) {
+      outOfDateCount++;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of '++'.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2347668677/issues/source?bid=3148782&fileBranchId=3216643#l175)